public/locales/en/translation.json

```diff
@@ -282,7 +286,6 @@
             "title": "Manual Sync Saves",
             "upload": "Upload"
         },
-        "maxRecentGames": "Played Recently to Show",
```

Comment:
weird that it is still removing a few keys not related with the PR. 🤔

Reply:
Ah, looks like it's caused by a couple deleted lines from here: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/commit/421ae61f5a1db85c83d6acac2f8601e515e9e452

Reply:
Yes, you are right. Let me try to fix those